Week1/database.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+let mysql = require('mysql');
+let connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'userdb'
```

Comment:
Using database 'userdb' is not necessary if you are going to create new_world database.